modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/edit_form/start.jsp

```diff
@@ -18,7 +18,14 @@ String fullName = namespace.concat(HtmlUtil.escapeAttribute(name));
 
 		<div aria-orientation="vertical" class="panel-group panel-group-flush" role="tablist">
 			<c:if test="<%= Validator.isNotNull(onSubmit) %>">
-				<div aria-label="<%= HtmlUtil.escape(portletDisplay.getTitle()) %>" class="input-container" role="group">
+				<c:choose>
+					<c:when test="<%= Validator.isNotNull(title) %>">
+						<div aria-label="<%= title %>" class="input-container" role="group">
+					</c:when>
+					<c:otherwise>
+						<div aria-label="<%= HtmlUtil.escape(portletDisplay.getTitle()) %>" class="input-container" role="group">
+					</c:otherwise>
```

Comment:
We can simplify this, and avoid code duplication, with something likee:
```jsp
<div aria-label="<%= HtmlUtil.escape(Validator.isNotNull(title) ? title : portletDisplay.getTitle()) %>" class="input-container" role="group">
```
Always escaping is not strictly required here, as `title` cannot be set by users, bit it doesn't hurt.